Makefile

```diff
@@ -22,13 +22,11 @@ install:
 	helm install connaisseur helm --atomic --create-namespace --namespace $(NAMESPACE)
 
 uninstall:
-	kubectl config set-context --current --namespace $(NAMESPACE)
-	helm uninstall connaisseur
+	helm uninstall connaisseur -n $(NAMESPACE)
 	kubectl delete ns $(NAMESPACE)
 
 upgrade:
-	kubectl config set-context --current --namespace $(NAMESPACE)
-	helm upgrade connaisseur helm --wait
+	helm upgrade connaisseur helm  -n $(NAMESPACE)--wait
```

Comment:
Do we also want to fix the NS in the integrations tests (ie https://github.com/sse-secure-systems/connaisseur/blob/master/tests/integration/integration-test.sh#L15 ) while you're at it?

Reply:
even introducing make as well to test both